Documentation/commands.md

```diff
@@ -317,6 +316,42 @@ Work in progress. Please contribute!
 
 Work in progress. Please contribute!
 
+### Logging
+
+By default, rkt will send logs directly to stdout/stderr, allowing them to be caputered by the invoking process.
+On host systems running systemd, rkt will attempt to integrate with journald on the host.
```

Comment:
It requires systemd >v44 on the host so I don't think it's worth it mentioning. In stage1 I guess we can say v219 and v220.